packages/flutter_tools/pubspec.yaml

```diff
@@ -23,6 +23,10 @@ dependencies:
   json_schema: 1.0.10
   linter: 0.1.74
   meta: 1.1.6
+  multicast_dns:
+    git:
+      url: https://github.com/dnfield/flutter_packages
```

Comment:
I assume this will change to flutter/packages or something once you migrate it?

Reply:
It can just come from pub.  Or we could pull it from flutter/packages - I'm just doing it this way so it can run through the CI tests for now.